core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/edit-ema-editor.component.ts

```diff
@@ -539,9 +545,19 @@ export class EditEmaEditorComponent implements OnInit, OnDestroy {
         };
     }): () => void {
         return (<Record<CUSTOMER_ACTIONS, () => void>>{
+            [CUSTOMER_ACTIONS.CONTENT_CHANGE]: () => {
+                this.store.updateEditorState(EDITOR_STATE.LOADED);
+            },
+
             [CUSTOMER_ACTIONS.SET_URL]: () => {
                 const payload = <SetUrlPayload>data.payload;
 
+                if (this.activatedRouter.snapshot.queryParams.url === payload.url) {
+                    this.store.updateEditorState(EDITOR_STATE.LOADED);
+                } else {
+                    this.store.updateEditorState(EDITOR_STATE.LOADING);
+                }
+
```

Comment:
Add comments here explaining why of this... we're going to forget about this in 2 days 🤣

Reply:
Done